source/_addons/duckdns.markdown

```diff
@@ -50,7 +50,8 @@ Use the following configuration in Home Assistant to use the generated certifica
 
 ```yaml
 http:
-  base_url: https://my-domain.duckdns.org:8123
+  base_url: my-domain.duckdns.org
+  server_port: 8123  # 8123 is also the default value
```

Comment:
We don't add optional/default values to examples.

Reply:
You are right. I have no clue why it didn’t work for me. Now it does...